doc/lsst.pipe.tasks/tasks/lsst.pipe.tasks.postprocess.TransformSourceTableTask.rst

```diff
@@ -0,0 +1,168 @@
+.. lsst-task-topic:: lsst.pipe.tasks.postprocess.TransformSourceTableTask
+
+########################
+TransformSourceTableTask
+########################
+
+
+``TransformSourceTableTask`` transforms the full source catalog
+(a ``source`` dataset) to a DPDD-specified Source Table (a ``sourceTable`` dataset).
+It extracts, transforms, renames, columns per a yaml specification, by default the
+`Source.yaml` in obs_package/policy.   Both inputs and outpus are per-detector.
+The input is typically a wide table and output a narrow table appropriate for
+concatenating into a per-visit table by ConsolidateSourceTableTask.
+
+It is the second of three postprocessing tasks to convert a `src` table to a
+per-visit Source Table that conforms to the standard data model.
+See ``ConsolidateSourceTableTask`` for the next step.
+
```

Comment:
You should also put in a link to the first step here.  You can in fact use the same boilerplate on all 3 tasks, "It is the <blank> step of three postprocessing steps.  The first is blah.  The second is blah.  The third is blah." and remove the bit that's redundant.